runtime/js/98_global_scope_shared.js

```diff
@@ -31,11 +31,67 @@ import * as messagePort from "ext:deno_web/13_message_port.js";
 import * as webidl from "ext:deno_webidl/00_webidl.js";
 import { DOMException } from "ext:deno_web/01_dom_exception.js";
 import * as abortSignal from "ext:deno_web/03_abort_signal.js";
-import * as imageData from "ext:deno_web/16_image_data.js";
 import { webgpu, webGPUNonEnumerable } from "ext:deno_webgpu/00_init.js";
 import * as webgpuSurface from "ext:deno_webgpu/02_surface.js";
 import { unstableIds } from "ext:runtime/90_deno_ns.js";
 
+const { op_lazy_load_esm } = core.ensureFastOps(true);
+let image;
+
+function ImageNonEnumerable(getter) {
+  let valueIsSet = false;
+  let value;
+
+  return {
+    get() {
+      loadImage();
+
+      if (valueIsSet) {
+        return value;
+      } else {
+        return getter();
+      }
+    },
+    set(v) {
+      loadImage();
+
+      valueIsSet = true;
+      value = v;
+    },
+    enumerable: false,
+    configurable: true,
+  };
+}
+function ImageWritable(getter) {
+  let valueIsSet = false;
+  let value;
+
+  return {
+    get() {
+      loadImage();
+
+      if (valueIsSet) {
+        return value;
+      } else {
+        return getter();
+      }
+    },
+    set(v) {
+      loadImage();
+
+      valueIsSet = true;
+      value = v;
+    },
+    enumerable: true,
+    configurable: true,
+  };
+}
+function loadImage() {
+  if (!image) {
+    image = op_lazy_load_esm("ext:deno_canvas/01_image.js");
+  }
+}
+
```

Comment:
quite a bit duplicate with webgpu. we should look into having a better solution that requires less setup for each lazy loaded file